src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -314,22 +313,18 @@ class AppDeployIntegrationTest
 
   test("kill all tasks of an App with scaling") {
     Given("a new app with multiple tasks")
-    val app = v2AppProxy(testBasePath / "tokill", "v1", instances = 2, withHealth = false)
+    val app = v2AppProxy(testBasePath / "tokill", "v1", instances = 1, withHealth = false)
```

Comment:
And we DID have errors like that in the past. I also would like it if you kept the count at 2.